### PR TITLE
[docs] Fix CORS section in installation

### PR DESCRIPTION
**Pull request by graceguo-supercat**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [x] Documentation

### SUMMARY
Fix the extra CORS Dependency in Superset installation step.

### TEST PLAN
https://superset.incubator.apache.org/installation.html#cors

### REVIEWERS
@michellethomas @mistercrunch @etr2460

### CODE REVIEW

**Review by etr2460 — APPROVED** [review]
doc change seems fine. I'm a bit confused why you'd need to separately install another dep instead of it being included in requirements or requirements-extra, but i don't have the historical context

**Comment by graceguo-supercat** [comment]
CORS is optional for Superset app. it's contributed by community: 
https://github.com/apache/incubator-superset/pull/478
I guess most of users/companies don't need to handle cross-domain issue.

**Comment by villebro** [comment]
What is the purpose of this change? If the purpose is to describe the process when not installing Superset via `pip`, then I think it might be better to explain when this alternative approach is needed.

**Comment by graceguo-supercat** [comment]
I feel confused by the current instruction. I am ok with all options .

**Before:**
Screen Shot 2020-01-14 at 1 19 50 PM

**After:**
Screen Shot 2020-01-14 at 1 26 18 PM

File: docs/installation.rst [hunk 1]
```diff
@@ -772,8 +772,9 @@ CORS
 
 The extra CORS Dependency must be installed:
 
-    superset[cors]
+.. code-block:: text
 
+    pip install -U flask-cors
```
[the remark this answers is not in the document]
Comment (graceguo-supercat) [reply docs/installation.rst]: Updated.

**Pull request merged**